.github/workflows/publish-release.yml

```diff
@@ -9,6 +9,12 @@ jobs:
   release-nugets:
     if: startsWith(github.ref, 'refs/tags/v')
     runs-on: ubuntu-latest
+    environment: |-
+      ${{
+         contains(github.event.release.tag_name, 'preview') && 'test'
+      || contains(github.event.release.tag_name, 'rc')      && 'staging'
+      ||                                                       'prod'
```

Comment:
Everything that is not `preview` or `rc` is production. How long before someone forgets our current standard and releases `alpha` or `beta`?

Reply:
I'm sure we can find a way to validate the tag name in the pipeline, but if not we will be documenting the release process. Not standardizing this is a non-starter anyways 😄

Reply:
Gave it a shot: https://github.com/Altinn/app-lib-dotnet/pull/903